router/core/context.go

```diff
@@ -266,6 +277,15 @@ func (c *requestContext) GetStringMapStringSlice(key string) (smss map[string][]
 	return
 }
 
+func (c *requestContext) ActiveSubgraph(subgraphRequest *http.Request) *Subgraph {
+	for _, sg := range c.subgraphs {
```

Comment:
We can do this once before we run the module handlers.

Reply:
Let's stick with that approach because we don't need to deal with concurrent access now. Every subgraph request has a different active subgraph.